comment/views/reactions.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseBadRequest, JsonResponse
+from django.shortcuts import get_object_or_404
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.http import require_POST
+
+from comment.models import Comment, ReactionInstance
+
+@require_POST
+@login_required
+def react(request, comment_id, reaction):
```

Comment:
Can you please use CBV instead of FBC? CBV is more clean choice.

Reply:
Do we really need a class based view? This function currently handles pretty basic stuff.

Reply:
Since version 1.4.0 all FBVs have been converted to CBVs. You can leave it as is and I may refactore it later.

Reply:
While I don't agree using a class based view will make the function any cleaner, in this case. I will convert it and commit the change in some time.